File: KODY/Podstawy/Hello.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

ROK_TERAZ = 2017
ROK_PY = 1991

def parzyste(n):
    ile = list(range(0, n+1, 2))
    print(ile)
    return len(ile)

def main(args):
    imie = input("Imię? ")
    wiek = int(input("ile masz lat? "))
    rok_urodzenia = ROK_TERAZ - wiek
    print ("witaj", imie)
    print ("Urodziłeś się w: ", ROK_TERAZ-wiek)
    
    if ROK_PY < rok_urodzenia:
        print("Jestem starszy")
    elif ROK_PY > rok_urodzenia:
        print("jestem młodszy")
    elif ROK_PY == rok_urodzenia:
        print("tyle samo lat")
    
    n = int(input("Podaj liczbę naturalną: "))
    print("Ilość parzystych: ", parzyste(n))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: KODY/Hello.cpp
/*
 * Hello.cpp
 * 
 */


#include <iostream>

using namespace std;

#define ROK_TERAZ 2017
#define ROK_CPP 1983

int parzysta(int n)
{
    int i = 0;
    for(i=0; i<=n; i+=2)
        cout << i << " ";
    return i / 2;
}
int main(int argc, char **argv)
{   
    char imie[20];
    int a = 0;
    int rok_urodzenia;
    
    cout << "imie?" << endl;
    cin.getline(imie, 20);
	cout << "hello" << imie << endl;
    cout << "Ile masz lat? ";
    cin >> a;
    rok_urodzenia = ROK_TERAZ - a;
    cout << "Urodziłeś się w: " << rok_urodzenia << endl;
    if(ROK_CPP < rok_urodzenia)
       cout << "Jestem starszy" << endl;
    else if(ROK_CPP  > rok_urodzenia)
        cout << "Jestem młodszy";
       
    int n;
    cout << "Podaj liczbę naturalną: ";
    cin >> n;
    cout << "Ilość parzystych: " << parzysta(n);
    
	return 0;
}

